src/main/java/bookstore/dto/BookDto.java

```diff
@@ -0,0 +1,8 @@
+package bookstore.dto;
+
+import java.math.BigDecimal;
+
+public record BookDto(Long id, String title,
+                      String author, String isbn,
+                      BigDecimal price, String description, String coverImage) {
```

Comment:
```suggestion
public record BookDto(Long id, 
                                        String title,
                                        String author, 
                                        String isbn,
                                        BigDecimal price, 
                                        String description, 
                                        String coverImage) {
```
this format is more readable